java/src/main/java/com/karhoo/demo/KarhooDemoApplication.java

```diff
@@ -0,0 +1,190 @@
+package com.karhoo.demo;
+
+import com.karhoo.demo.model.*;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.ArrayList;
+
+@SpringBootApplication
+public class KarhooDemoApplication {
+
+    public static String BOOKING_URL = "/v1/bookings";
+    public static String BOOKING_STATUS_URL = "/v1/bookings/%s/status";
+    public static String BOOKING_HISTORY_URL = "/v1/bookings/%s/history";
+    public static String FINAL_FARE_URL = "/v1/fares/trip/%s";
+    public static String FOLLOW_BOOKING_URL = "/v1/bookings/follow/%s";
+    public static String QUOTES_REQUEST_URL = "/v1/quotes";
+    public static String QUOTES_LIST_URL = "/v1/quotes/%s";
+    public static String TRACK_DRIVER_URL = "/v1/bookings/%s/track";
+    public static int MAX_WAIT_TIME = 3000;
+    ;
+
+    public static void main(String[] args) throws Exception {
+
+        SpringApplication.run(KarhooDemoApplication.class, args);
+        /**
+         * Step 1 - Choosing the correct environment and configuration setup.
+         * In this sample the host and scheme are set in KarhooApiClient
+         * Karhoo provides a Sandbox environment that you should use for testing the integration and a
+         * Production environment that can be used when you go live.
+         * The Sandbox environment is available at `https://rest.sandbox.karhoo.com`
+         * The Production environment is available at `https://rest.karhoo.com`
+         * Karhoo will provide you with these credentials.
+         * Credentials for Production should be stored securely on your server
+         *
+         * Step 2 - Authenticate and retrieve a JWT and refresh token
+         **/
+        String username = "Sean.keane@karhoo.com";
```

Comment:
Can you please remove the values and use [USERNAME_PLACEHOLDER] (same for the password) ... it can be covered in the readme